diff --git a/src/google/protobuf/CMakeLists.txt b/src/google/protobuf/CMakeLists.txt
index 481d213..b7e326c 100644
--- a/src/google/protobuf/CMakeLists.txt
+++ b/src/google/protobuf/CMakeLists.txt
@@ -1,6 +1,6 @@
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
 
-set(SGXSSL_INC /opt/intel/sgxssl/include)
+set(SGXSSL_INC ${SGX_SSL_DIR}/include)
 
 # ${CMAKE_CURRENT_SOURCE_DIR}/util/time_util.cc
 set(PROTOBUF_SGX_SRCS
